update.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e # everything must pass
 
-python .prerequisites.py "$@"
+python3 .prerequisites.py "$@"
```

Comment:
Nice catch, I overlooked this in the recent py2-removal. 

On Ubuntu at least `python` will default to `python2`. I wouldn't be surprised if osx did the same. I run Arch and `python` points at the most recent version of python installed:

```
$ python --version
Python 3.10.1
```